### PR TITLE
refactor: dark mode article

### PR DESCRIPTION
**Pull request by vchimev**
Also, reflect changes in https://github.com/NativeScript/NativeScript/pull/8084.

### CODE REVIEW

File: docs/ui/dark-mode.md [hunk 1]
```diff
@@ -84,12 +86,30 @@ To illustrate this behavior, the sample CSS styles below will paint the Pages of
...
 
+{% endnativescript %}
+
+{% angular %}
+
+To illustrate this behavior, the sample CSS styles below will paint the elements with assigned `.my-class` class of your application in *light blue* color on Light system appearance and in *dark blue* on dark system appearance:
+
+```css
+.my-class {
+    background-color: lightgreen;
```
Comment (elena-p) [inline docs/ui/dark-mode.md]: blue or green? :)

Comment (vchimev) [reply docs/ui/dark-mode.md]: Let it be `blue`. :))

File: docs/ui/dark-mode.md [hunk 2]
```diff
@@ -84,12 +86,30 @@ To illustrate this behavior, the sample CSS styles below will paint the Pages of
...
+
+To illustrate this behavior, the sample CSS styles below will paint the elements with assigned `.my-class` class of your application in *light blue* color on Light system appearance and in *dark blue* on dark system appearance:
+
+```css
+.my-class {
+    background-color: lightblue;
+}
+
+:host-context(.ns-dark) .my-class {
+    background-color: lightblue;
```
Comment (elena-p) [inline docs/ui/dark-mode.md]: lightblue -> darkblue

**Review by elena-p — APPROVED** [review]

File: docs/ui/dark-mode.md (hunk 2 above)
Comment (vchimev) [reply docs/ui/dark-mode.md]: image

**Pull request merged**